client/src/Q&As/Styles.js

```diff
@@ -3,14 +3,61 @@ import styled, {css} from 'styled-components';
 
 
 export const IndexGrid = styled.div`
+  overflow: scroll;
   padding: 5px;
   font-family: Sans-Serif;
   font-size: 18px;
   border: 1px solid grey;
   display: grid;
+  grid-template-columns: 2fr 266px;
+  grid-template-rows: 100px 300px 100px;
+  gap: 10px 0px;
+  grid-template-areas:
+    "search ."
+    "question ."
+    "buttons-buttons .";
+  color: #6f67bf;
+`;
+
+export const QuestionsContainer = styled.div.attrs({className: 'Questions_container'})`
+  overflow: scroll;
```

Comment:
This line makes every question scrollable. You can create a div that holds all of the questions/answers, then make that scrollable. That way only the q/as move and the search bar and buttons will stay visible.

Reply:
ah okay I'll try that! I'm still working on it 😖